ui/src/components/easter/hero/HeroContainer.js

```diff
@@ -1,3 +1,6 @@
+import 'react-modal-video/scss/modal-video.scss';
```

Comment:
also, you don't have to fix it for this branch, but please always make sure that your code is formatted properly. there's a lot of whitespaces and that makes it very difficult for people (including your future self) to understand it
![Screenshot 2022-03-18 at 20 40 57](https://user-images.githubusercontent.com/33679483/159004683-32aeb798-b510-4b84-a4b3-d19ecaf03223.png)